src/pages/studyView/StudyViewPage.tsx

```diff
@@ -842,15 +842,13 @@ export default class StudyViewPage extends React.Component<
                                                     .driverAnnotationSettings
                                             }
                                             studyIdToStudy={
-                                                this.store.studyIdToStudy.result
+                                                this.store.studyIdToStudy
                                             }
                                             structuralVariants={
                                                 this.store.structuralVariants
-                                                    .result
                                             }
-                                            hugoGeneSymbols={
+                                            allHugoGeneSymbols={
```

Comment:
Hi Bryan, sorry it took me SO long to respond.  This is quite confusing to have a property hugeGeneSymbols AND allHugeGeneSymbols.  Lets think if there's some better we can handle this.

Reply:
The basic problem is that we need to wait for the gene symbols to load but we only want to do that when the tab is actually showing.  We can handle that more elegantly with some kind of loading wrapper.  Maybe we can do a pair programming session and figure it out.